### DISCUSSION TITLE
How to make the make calendar week start from Sunday?

### DISCUSSION THREAD

**Discussion by IgorLab19k** [wojtekmaj/react-calendar#721] [Q&A] [answered]
Hello, my question is how to make calendar week days start from SUN (Sunday) instead of Monday. I`m sure it` s possible, but can`t find proper props.

PS. image 1 - my view; image 2 - desired view;

![image](https://user-images.githubusercontent.com/119309514/209577786-502c4172-2330-4151-87d9-12165991947c.png)
 
![image](https://user-images.githubusercontent.com/119309514/209577766-11fc31fe-46e8-4fe9-8b94-a5b4fc17f98c.png)

**Comment by wojtekmaj** [comment, marked as answer]
Check `calendarType` and `locale` in documentation. I would recommend checking out `locale` first and only use `calendarType` override if needed.

**Reply by lcim** [reply]
Sorry I am not able to do what you recommended. I saw a post that edited localizer:
const localizer = dateFnsLocalizer({
    ...,
    startOfWeek: () => startOfWeek( new Date(), {weekStartsOn: 0}),
    ...,
})
This solved my problem but I wish to also be able to solve it the way you recommended. Could you provide a link to the documentation you mean please.
Thanks so much for your wonderful works.
